src/Symfony/Component/Filesystem/Tests/FilesystemTestCase.php

```diff
@@ -27,17 +27,43 @@ class FilesystemTestCase extends \PHPUnit_Framework_TestCase
      */
     protected $workspace = null;
 
-    private static $symlinkOnWindows = null;
+    /**
+     * @var null Flag for hard links on Windows
+     */
+    protected static $linkOnWindows = null;
```

Comment:
Fixed in 584cafe

I did not remove the phpdoc, the other properties have doc so I rather fixed this one to say `null|bool` to give correct hints for IDE's.